HostedServices/ScheduledEventHostedService.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using snipetrain_bot.Services;
+
+namespace snipetrain_bot.HostedServices
+{
+    public class ScheduledPartyHostedService : IHostedService, IDisposable
+    {
+
+        private Timer _timer;
+        private readonly ILogger<ScheduledPartyHostedService> _logger;
+        private readonly IPartyService _partyService;
+        private readonly DiscordRunner _runner;
+
+        public ScheduledPartyHostedService(IPartyService partyService, DiscordRunner runner)
+        {
+            _partyService = partyService;
+            _runner = runner;
+        }
+
+        public Task StartAsync(CancellationToken stoppingToken)
+        {
+
+            var seconds = 60;
+            _timer = new Timer(DoWork, null, TimeSpan.Zero, TimeSpan.FromSeconds(seconds));
+
+            return Task.CompletedTask;
+        }
+
+        private async void DoWork(object state)
+        {
+            var partyList = await _partyService.GetPartiesAsync();
+            var ChannelNA = ulong.Parse("700712690288427129");
+            var ChannelEU = ulong.Parse("747139803711012884");
+            var roleId = "272275923208896512";
+            
+
+            foreach (var party in partyList)
+            {
+                var DateCompVal = party.EventDay.CompareTo(DateTime.Now);
```

Comment:
This timer runs every minute, meaning that if the date of `party.EventDay` has seconds precision and they are not the exact same as when the time runs, this won't pick it up. You might want to save to a minute precision and compare that.